Acadian.FSharp/FSharp.fs

```diff
@@ -10,6 +10,24 @@ module Prelude =
     /// Flips the order of two arguments of a function. `flip f a b = f b a`
     let inline flip f a b = f b a
 
+    /// Performs a side effect and returns the input of said side effect.
+    let inline tee f x = f x |> ignore; x
+
+    /// Custom operator for `tee`: Given a value, apply a function to it, ignore the result, then return the original value.
```

Comment:
IMO the description here and the one on tee itself should match. I like something in between the two: "Applies a side-effect function to a value, ignoring the result, and returns the original value."